web/styles/app_components.css

```diff
@@ -1022,13 +1022,22 @@ div.overlay {
     align-items: center;
     justify-content: space-between;
     outline: none;
+    color: var(--color-text-default);
 
     .fa-chevron-down {
         position: relative;
         top: -1px;
     }
 }
 
+.setting-disabled-option {
+    color: hsl(38deg 46% 54%);
+
+    & i {
+        padding: 0 5px 0 1px;
```

Comment:
Can you document the rationale for these values?

Reply:
Done.